Makefile.docker

```diff
@@ -0,0 +1,95 @@
+REPO := logdna-agent-v2
+
+DOCKER := docker
+DOCKER_PRIVATE_IMAGE := us.gcr.io/logdna-k8s/logdna-agent-v2
+DOCKER_PUBLIC_IMAGE := docker.io/logdna/logdna-agent
+DOCKER_IBM_IMAGE := icr.io/ext/logdna-agent
+
+# The image repo and tag can be modified e.g.
+# `make -f Makefile.docker build IMAGE=docker.io/rust:1.42.0`
+IMAGE_REPO ?= docker.io/rust
+IMAGE_TAG ?= 1.42
+IMAGE ?= $(IMAGE_REPO):$(IMAGE_TAG)
+IMAGE := $(IMAGE)
+
+VCS_REF := $(shell git rev-parse --short HEAD)
+VCS_URL := https://github.com/logdna/logdna-agent-v2
```

Comment:
Wanted to use a web friendly link instead of the github link which in my cases was a git ssh link